pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_22,
+  kwin,
+  kpackage,
+  p7zip,
+  git,
+}:
+buildNpmPackage rec {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchFromGitHub {
+    owner = "anametologin";
+    repo = pname;
+    rev = version;
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
can i ask what this does? more precise?

Reply:
Yep. Without this version could technically also refer to a branch named the same.